apps/ledger-live-mobile/src/components/RequiresCameraPermissions/Fallback.tsx

```diff
@@ -27,6 +26,9 @@ const FallbackCameraBody: React.FC<Props> = ({
 }: Props) => {
   const { colors } = useTheme();
 
+  // benefit from component scope colors variable for IconSettings
+  const IconSettings = () => <Icon name="settings" size={16} color={colors.constant.white} />
+
```

Comment:
Could you fix the prettier formatting, please?

Reply:
yep sure, i'm using the default prettier config in my VSCode.

Reply:
can you try using neutral.c100 please? :) Instead of using osTheme 
if neutral.c100 not working try neutral.c00 I don't remember de right one 
it will convert automatically according to the current theme

Reply:
sure ! can you please send me (if exists) theming documentation or color palette docs/ref ?

Reply:
@mcayuelas-ledger done ✅